cardano-api/src/Cardano/Api/LedgerEvent.hs

```diff
@@ -153,23 +155,33 @@ pattern LEMirTransfer rp tp rtt ttr <-
         ( MirEvent
             ( MirTransfer
                 ( InstantaneousRewards
-                    (convertSumRewardsMap -> rp)
-                    (convertSumRewardsMap -> tp)
+                    (convertSumRewardsMapSimple -> rp)
+                    (convertSumRewardsMapSimple -> tp)
                     (fromShelleyDeltaLovelace -> rtt)
                     (fromShelleyDeltaLovelace -> ttr)
                   )
               )
           )
       )
 
-convertSumRewardsMap ::
+convertRewardEventMapNew ::
```

Comment:
Can we avoid these names? `New` and `Simple` are completely useless and tell me nothing about the difference between these